drivers/usb/device/usb_dc_stm32.c

```diff
@@ -305,6 +305,10 @@ static int usb_dc_stm32_clock_enable(void)
 #if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_usbphyc)
 	LL_AHB1_GRP1_EnableClock(LL_AHB1_GRP1_PERIPH_OTGHSULPI);
 	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_OTGPHYC);
+#elif defined(CONFIG_SOC_SERIES_STM32H7X)
+	/* Disable ULPI interface (for external high-speed PHY) clock */
+	LL_AHB1_GRP1_DisableClock(LL_AHB1_GRP1_PERIPH_USB1OTGHSULPI);
+	LL_AHB1_GRP1_DisableClockSleep(LL_AHB1_GRP1_PERIPH_USB1OTGHSULPI);
 #else
 	/* Disable ULPI interface (for external high-speed PHY) clock */
 	LL_AHB1_GRP1_DisableClock(LL_AHB1_GRP1_PERIPH_OTGHSULPI);
```

Comment:
For my information, is the disable step required ?

Reply:
Thanks for the heads up.
Can we then keep only the disable in sleep mode ?

Reply:
I have just pushed a new version with a commit removing the corresponding lines.